kerchunk/combine.py

```diff
@@ -672,9 +672,9 @@ class with ``translate()`` method
 
     # make delayed calls
     tasks = [single_task(u) for u in urls]
-    tasks_per_batch = len(tasks) // n_batches
+    tasks_per_batch = -(-len(tasks) // n_batches)
```

Comment:
why minus minus? This rounds up?

Reply:
Yup, I think for the way this loop is now written, we want to round up. E.g. for 3 tasks and 2 batches we'd want `tasks_per_batch` = 2